### PR TITLE
SUS-4194 in Answers extension use getLocalUrl instead manual creation

### PR DESCRIPTION
**Pull request by sqreek**
https://wikia-inc.atlassian.net/browse/SUS-4194

@Wikia/sus

### CODE REVIEW

**Review by macbre — APPROVED** [review]

**Review by fraszczakszymon — CHANGES_REQUESTED** [review]
What about 
https://github.com/Wikia/app/blob/dev/extensions/wikia/Answers/Answers.php#L236
https://github.com/Wikia/app/blob/dev/extensions/wikia/Answers/Answers.php#257
https://github.com/Wikia/app/blob/dev/extensions/wikia/Answers/SpecialGetQuestionWidget.php#L44
https://github.com/Wikia/app/blob/dev/extensions/wikia/Answers/SpecialGetQuestionWidget.php#L46

File: extensions/wikia/Answers/HomePageList.php [hunk 1]
```diff
@@ -277,4 +277,19 @@ function _homepage_questions($key, $title, $ignore_cache = false) {
 		wfProfileOut(__METHOD__);
 		return $html;
 	}
+
+	/**
+	 * @param string $pageTitle
+	 * @param string $text
+	 * @return string
+	 */
+	private function createLinkListElement($pageTitle, $text): string
```
Comment (mszabo-wikia) [inline extensions/wikia/Answers/HomePageList.php]: Let's use [MW coding conventions](https://www.mediawiki.org/wiki/CC/PHP) - there's also an [XML configuration snippet](https://www.mediawiki.org/wiki/JetBrains_IDEs#MediaWiki_code_style) for IDEA

**Review by mszabo-wikia — CHANGES_REQUESTED** [review]

**Comment by sqreek** [comment]
@fraszczakszymon wgServer will handle language path and answer.wikia.com is a specific wikia that we will rename from http to https by grepping whole repo

**Review by mszabo-wikia — APPROVED** [review]

**Review by mszabo-wikia — DISMISSED** [review]
the `wgServer` won't contain path components, it will have to be concatenated with `wgScriptPath` and friends

**Comment by sqreek** [comment]
@fraszczakszymon @TK-999 problematic code was removed

**Review by mszabo-wikia — APPROVED** [review]

**Comment by fraszczakszymon** [comment]
@sqreek I like this approach :D

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**